package.json

```diff
@@ -9,76 +9,78 @@
     "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",
     "nx": "nx",
     "start": "nx serve",
-    "build": "nx run-many --target=build --configuration=production --all --parallel",
+    "build": "nx run-many --target=build --configuration=production --all",
     "test": "nx run-many --target=test --all --parallel",
     "lint": "nx workspace-lint && nx run-many --target=lint --all --parallel",
-    "e2e": "nx run-many --target=e2e --all",
+    "e2e": "nx run-many --target=e2e --all --parallel",
     "affected:apps": "nx affected:apps",
     "affected:libs": "nx affected:libs",
     "affected:build": "nx affected:build",
-    "affected:e2e": "nx affected:e2e",
-    "affected:test": "nx affected:test",
-    "affected:lint": "nx affected:lint",
+    "affected:e2e": "nx affected:e2e --parallel",
+    "affected:test": "nx affected:test --parallel",
+    "affected:lint": "nx affected:lint --parallel",
     "affected:dep-graph": "nx affected:dep-graph",
     "affected": "nx affected",
     "format": "nx format:write && pnpx prettier --config .prettierrc --write \"**/*.*\" \"!dist/**\" \"!pnpm-lock.yaml\"",
     "format:write": "nx format:write",
     "format:check": "nx format:check",
-    "update": "nx migrate latest",
+    "update": "nx migrate @nrwl/worskpace && nx migrate latest",
     "workspace-generator": "nx workspace-generator",
     "dep-graph": "nx dep-graph",
     "help": "nx help",
-    "sort-projects": "json -I -f workspace.json -e \"this.projects = Object.fromEntries(Object.entries(this.projects).sort(([a], [b]) => (a < b) ? -1 : (a > b) ? 1 : 0));\" && json -I -f nx.json -e \"this.projects = Object.fromEntries(Object.entries(this.projects).sort(([a], [b]) => (a < b) ? -1 : (a > b) ? 1 : 0));\""
+    "sort-projects": "json -I -f workspace.json -e \"this.projects = Object.fromEntries(Object.entries(this.projects).sort(([a], [b]) => (a < b) ? -1 : (a > b) ? 1 : 0));\" && json -I -f nx.json -e \"this.projects = Object.fromEntries(Object.entries(this.projects).sort(([a], [b]) => (a < b) ? -1 : (a > b) ? 1 : 0));\"",
```

Comment:
I think someone replied to one of your Twits saying that this may be simplified with `(a, b) => a - b`. I think there is a problem with that in Javascript (type conversion problem or something like that). 

Was that the reason you didn't use that solution?

Reply:
That only works for numbers, not strings, which he also said in the Angular Discord server. He then suggested [String.prototype.localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) which would make it `.sort(([a], [b]) => a.localeCompare(b))`

Reply:
Switched to the suggested implementation.